docs/product-installation-guide/src/main/asciidoc/clustering-con.adoc

```diff
@@ -1,9 +1,20 @@
 [id='clustering-con']
-= Clustering with {PRODUCT} for design-time development environments
+= Clustering with {PRODUCT} 
+ifdef::DM[]
+for design-time development environments
+endif::[]  
+
+COMMENT: Need to revise and explain the differences between clustering for development and clustering for runtime.
 
 [NOTE]
 ====
-This section is specifically intended for {CENTRAL} development environments where you want to cluster the Git repository. It is not necessary to create a clustered environment for {KIE_SERVER} production environments.
+This section is specifically intended for {CENTRAL} 
+ifdef::DM[]
+development
+endif::[]  
+environments where you want to cluster the Git repository. It is not necessary to create a clustered environment for {KIE_SERVER} production environments.
```

Comment:
not sure what you mean by "it's not necessary to create a clustered environment for process server"...

it is valid configuration but requires less steps